src/applications/disability-benefits/2346/containers/App.jsx

```diff
@@ -1,12 +1,27 @@
 import React from 'react';
-
+import Breadcrumbs from '@department-of-veterans-affairs/formation-react/Breadcrumbs';
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
 export default function App({ location, children }) {
   return (
-    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      {children}
-    </RoutedSavableApp>
+    <>
+      <Breadcrumbs>
+        {[
```

Comment:
You can actually remove the wrapping `{[ ]}` and remove the key properties on all of the children! 🙂

Reply:
Also we'll need `hrefs` here as well as `rel="noreferrer noopener"` for links that are not va.gov based 📈